### PR TITLE
Scaffold Logs and Metrics management

### PR DESCRIPTION
**Pull request by Dermah**
Outline the methods for pushing metrics and logging messages. Will enable us to log to multiple places down the road

- [x] Document
- [x] ~~Make/adjust issues~~

### CODE REVIEW

File: src/metrics/ConsoleLogger.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// ConsoleLogger simply prints log messages to the console.
+// Only really useful if you're running the backup manager manually
+
+let log = () => {
+
```
Comment (mrjackdavis) [inline src/metrics/ConsoleLogger.js]: Should we just implement this now?

File: src/metrics/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class MetricManager {
```
Comment (mrjackdavis) [inline src/metrics/index.js]: Hmm, quite a bit of implementation required here... (notsureifgusta)

File: src/metrics/ConsoleLogger.js (hunk 1 above)
Comment (Dermah) [inline src/metrics/ConsoleLogger.js]: Yeah...

File: src/metrics/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import winston from 'winston';
+
+// Log a message to all logging handlers
+let log = (message) => {
```
Comment (mrjackdavis) [inline src/metrics/index.js]:
```
export let log = (message) => {
```
nicer style?

**Comment by mrjackdavis** [comment]
Approved! Only nit-picky comments. Happy to merge

**Pull request merged**